src/styles/messages.scss

```diff
@@ -16,69 +16,77 @@
 
 @import './common/vars';
 
-.message-card {
-	@include status-card;
+.message-card-wrapper {
+	padding-bottom: 5px;
+	margin-right: 10px;
 
-	// lb-count - number of displaying labels - to make offset for name and session
-	--lb-count: 0;
-	// lb-count variable assignment through the custom html property
-	&[data-lb-count='0'] {
-		--lb-count: 0;
-	}
-	&[data-lb-count='1'] {
-		--lb-count: 1;
-	}
-	&[data-lb-count='2'] {
-		--lb-count: 2;
+	&:hover .message-card {
+		background-color: #f3fafc;
```

Comment:
This should probably be extracted to colors.scss as well

Reply:
done: https://github.com/th2-net/th2-rpt-viewer/commit/907beacefad62a9016631d37f6b54a5708e93ca4